load/UpdateDBPBiblesTable.py

```diff
@@ -302,10 +300,6 @@ def bibleTextCopyright(self, bibleId, languageRecords):
 
 	def testBibleForeignKeys(self, bibleId):
 		hasErrors = False
-		hasErrors |= self.testDeleteInTable(bibleId, self.config.database_user_db_name, "user_bookmarks")
```

Comment:
@bradflood The initial proposal suggested removing the invocation to the `testBibleForeignKeys` method. However, I have decided to keep the invocation to this method while removing all references to the `dbp_users` entities. I made this decision because the method also uses the bible_translations entity to verify if the `bibleId` can be deleted. It is important to note that the `bible_translations` entity belongs to in the `dbp_NEWDATA` database